src/Analyzers/CSharp/CodeFixes/ConvertNamespace/ConvertNamespaceTransform.cs

```diff
@@ -89,7 +89,7 @@ public static async Task<(Document document, TextSpan semicolonSpan)> ConvertNam
                 return null;
 
             // Auto-formatting options are not relevant since they only control behavior on typing.
-            var indentationOptions = new IndentationOptions(options, AutoFormattingOptions.Default);
+            var indentationOptions = new IndentationOptions(options);
```

Comment:
This might use some more cleanup, but I'd rather leave that for a follow up.

`IndentationOptions` comprise of three components: `SyntaxFormattingOptions`, `AutoFormattingOptions` and smart indent kind.

The indentation service has a single entry point that takes `IndentationOptions`, which is used by both code that is indenting while typing and "offline" code fixes. In the former case the `AutoFormattingOptions` are relevant and need to be passed in, while in the latter they are not applicable. ConvertNamespace code fix does fetch `SyntaxFormattingOptions` from the provider as you'd expect. Here it just needs to wrap them in `IndentationOptions` so that it call the indentation service. Ideally, we would have a separate end point for indentation while typing and otherwise.